app/controllers/pages_controller.rb

```diff
@@ -35,4 +35,12 @@ def beta?
   end
   helper_method :beta?
 
+  # Renders keywords to the page for content writers to inspect and use as a guide
+  # Gracefully falls back to the page's path if no keywords are specified
+  # to help reduce content workload
+  def content_keywords
+    @page.keywords || request.path.split("/").reject(&:empty?).map { |segment| segment.gsub("-", " ") }.join(", ")
```

Comment:
Feels like this logic should live inside the page model so that we don't need the additional view helper and the view can call `@page.keywords`.

Reply:
`request` isn't the concern of the page model so I've added the fallback logic as a helper method in the controller.

Reply:
I believe you should have access to the request object in the page model via `view_context`.

https://github.com/buildkite/docs/blob/a82878cda347d6de31ed6c357d53e11d81733845/app/controllers/pages_controller.rb#L12

Which means you can do something like the following in `page.rb`:

```ruby
@view.request.path
```

Reply:
hi @dannymidnight thanks for the tip, I've made the change:

```ruby
def keywords
  # Gracefully falls back to the page's path if no keywords are specified to help reduce technical writer workload
  front_matter.fetch(:keywords) || keywords_from_path
end
```

Also made a few minor refactoring. Please see latest commits. Cheers :)